index.js

```diff
@@ -15,14 +15,37 @@ const {
 const quickInterval = QUICK_NAP_DURATION_SECONDS * 1000,
     slowInterval = SLOW_NAP_DURATION_MINUTES * 60_000
 
+const {
+    readSettings,
+    checkForFocusBearInstall,
+} = require('./src/applescript/fs.js')
+
+const {
+    setCalsToExclude,
+    setEventsToExclude,
+} = require('./src/applescript/calendar.js')
+
+const { setNagState } = require('./src/intention.js')
+
+async function setSettings() {
+    const obj = await readSettings()
```

Comment:
is this needed?

Reply:
If you mean the obj declaration, nope, that was leftover from testing. Removed in the latest commit.